servicenow-instance-wakeup.go

```diff
@@ -88,6 +88,8 @@ func wakeUpInstance(username string, password string) chromedp.Tasks {
 		chromedp.Navigate(`https://developer.servicenow.com/ssologin.do?relayState=%2Fapp.do%23%21%2Finstance%3Fwu%3Dtrue`),
 		chromedp.WaitVisible(`.logo`),
 		chromedp.SendKeys(`#username`, username, chromedp.ByID),
+		chromedp.Click(`#usernameSubmitButton`, chromedp.ByID),
+		chromedp.WaitVisible(`.logo`),
```

Comment:
This what you have works too, since the password field is already filled, but a better way would be to wait for the element with the id `password` to be visible, something similar:
 ```suggestion
		chromedp.WaitVisible(`#password`),
```
Could you rewrite it please?